src/App.js

```diff
@@ -1,10 +1,34 @@
-import React from "react";
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+// Components
+import Home from './AppComponents/Home';
+import OrderConfirmation from './AppComponents/OrderConfirmation';
+import PizzaForm from './AppComponents/PizzaForm';
+import HeaderNav from './HeaderNav';
 
 const App = () => {
+  const [orders, setOrders] = useState([]);
+
+  const addToOrders = (inputOrder) => {
+    setOrders([...orders, inputOrder]);
+  };
+
+  console.log(orders);
   return (
     <>
-      <h1>Lambda Eats</h1>
-      <p>You can remove this code and create your own header</p>
+      <Router>
+        <HeaderNav />
+        <Switch>
+          <Route path="/orderConfirmation">
+            <OrderConfirmation order={orders[orders.length - 1]} />
+          </Route>
+          <Route path="/pizza">
+            <PizzaForm addToOrders={(input) => addToOrders(input)} />
+          </Route>
+          <Route path="/" component={Home} />
+        </Switch>
```

Comment:
Instead of using the `<Switch>` component you can using `exact path="/"` since none of the other routes are nested. 

Here's some more info on the switch component. https://medium.com/@jenniferdobak/react-router-vs-switch-components-2af3a9fc72e

Reply:
Ah makes sense. I always thought it was needed.